ui/src/stories/InfiniteScroll/InfiniteScroll.stories.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Story } from '@storybook/react';
+import InfiniteScroll, { Props } from 'core/components/InfiniteScroll';
+import { useState } from 'react';
+import Styled from './Styled';
+
+export default {
+  title: 'Components/InfiniteScroll',
+  component: InfiniteScroll,
+  parameters: { controls: { sort: 'requiredFirst' } },
+  argTypes: {
+    loadMore: {
+      description: 'This function returns new items to be shown in the list.',
```

Comment:
i think the description should be:
```suggestion
      description: 'This function will be called as long as hasMore is true and when scroll is at the end.',
```